extensions/amp-accordion/1.0/accordion.js

```diff
@@ -331,25 +390,14 @@ export function AccordionSection({
   }, [expanded, animate]);
 
   return (
-    <Comp {...rest} expanded={expanded}>
-      <HeaderComp
-        role="button"
-        className={`${headerClassName} ${classes.sectionChild} ${classes.header}`}
-        aria-controls={contentId}
-        tabIndex="0"
-        onClick={expandHandler}
-        aria-expanded={String(expanded)}
-      >
-        {header}
-      </HeaderComp>
-      <ContentComp
-        ref={contentRef}
-        className={`${contentClassName} ${classes.sectionChild} ${classes.content}`}
-        id={contentId}
-        hidden={!expanded}
-      >
-        {children}
-      </ContentComp>
+    <Comp
+      {...rest}
+      ref={contentRef}
```

Comment:
This also needs `aria-labelledby` and `role=region`, but if you want to share `headerId` in a separate PR feel free to add to the tracking item.

Reply:
Thanks, yep, this is tracked in the tracker https://github.com/ampproject/amphtml/issues/30445.  I'll address in a separate PR!